src/Text/LLVM/PP.hs

```diff
@@ -757,6 +757,7 @@ ppValue' pp val = case val of
   ValZeroInit        -> "zeroinitializer"
   ValAsm s a i c     -> ppAsm s a i c
   ValMd m            -> ppValMd' pp m
+  ValBlockAddr a b   -> ppTyped (ppValue' pp) a <> pp b
```

Comment:
Wouldn't this need to print as `blockaddress(a, b)`, like it does for `ConstBlockAddr`?

https://github.com/elliottt/llvm-pretty/blob/641ae933652369cf0d7e1e47c8928f631fb71858/src/Text/LLVM/PP.hs#L838